subsys/bluetooth/mesh/shell.c

```diff
@@ -1932,6 +2350,128 @@ static int cmd_provision_adv(const struct shell *shell, size_t argc,
 
 	return 0;
 }
+
+static int cmd_prov_close_link(const struct shell *shell, size_t argc,
+			     char *argv[])
+{
+	pb_adv.link_close(PROV_BEARER_LINK_STATUS_FAIL);
```

Comment:
Shouldn't bypass the provisioning API like this. Add a function to prov.h instead, so we don't get a bug here when we add PB-GATT